tree/ntuple/v7/inc/ROOT/RPageStorage.hxx

```diff
@@ -460,6 +460,8 @@ public:
    std::uint64_t CommitCluster(NTupleSize_t nEntries) final;
    void CommitClusterGroup() final;
    void CommitDatasetImpl() final;
+
+   RPage ReservePage(ColumnHandle_t columnHandle, std::size_t nElements) final;
```

Comment:
Can we move this even higher in `RPageSink` (and then still override it in `RPageSinkBuf`)? AFAICT `fPageAllocator` is in `RPageStorage`

Reply:
Good idea!